test/client/spec/components/victory-scatter/victory-scatter.spec.jsx

```diff
@@ -2,68 +2,73 @@
-      // default width and height
-      expect(svg.style.width).to.equal(`${VictoryScatter.defaultProps.width}px`);
-      expect(svg.style.height).to.equal(`${VictoryScatter.defaultProps.height}px`);
+      const wrapper = shallow(
+        <VictoryScatter/>
+      );
+      const svg = wrapper.find("svg");
+      expect(svg.prop("style").width).to.equal(VictoryScatter.defaultProps.width);
+      expect(svg.prop("style").height).to.equal(VictoryScatter.defaultProps.height);
     });
   });
 
   describe("rendering data", () => {
     it("renders points for {x, y} shaped data (default)", () => {
       const data = _.range(10).map((i) => ({x: i, y: i}));
-      renderedComponent = TestUtils.renderIntoDocument(<VictoryScatter data={data}/>);
-      const path = TestUtils.scryRenderedDOMComponentsWithTag(renderedComponent, "path");
-      expect(path.length).to.equal(10);
+      const wrapper = shallow(
+        <VictoryScatter data={data}/>
+      );
+      const points = wrapper.find(Point);
+      expect(points.length).to.equal(10);
     });
 
     it("renders points for array-shaped data", () => {
       const data = _.range(20).map((i) => [i, i]);
-      renderedComponent = TestUtils.renderIntoDocument(<VictoryScatter data={data} x={0} y={1}/>);
-      const path = TestUtils.scryRenderedDOMComponentsWithTag(renderedComponent, "path");
-      expect(path.length).to.equal(20);
+      const wrapper = shallow(
+        <VictoryScatter data={data} x={0} y={1}/>
+      );
+      const points = wrapper.find(Point);
+      expect(points.length).to.equal(20);
     });
 
     it("renders points for deeply-nested data", () => {
       const data = _.range(40).map((i) => ({a: {b: [{x: i, y: i}]}}));
-      renderedComponent = TestUtils.renderIntoDocument(
+      const wrapper = shallow(
         <VictoryScatter data={data} x="a.b[0].x" y="a.b[0].y"/>
       );
-      const path = TestUtils.scryRenderedDOMComponentsWithTag(renderedComponent, "path");
-      expect(path.length).to.equal(40);
+      const points = wrapper.find(Point);
+      expect(points.length).to.equal(40);
     });
 
     it("renders data values with null accessor", () => {
       const data = _.range(30);
-      renderedComponent = TestUtils.renderIntoDocument(
+      const wrapper = shallow(
         <VictoryScatter data={data} x={null} y={null}/>
       );
-      const path = TestUtils.scryRenderedDOMComponentsWithTag(renderedComponent, "path");
-      expect(path.length).to.equal(30);
+      const points = wrapper.find(Point);
+      expect(points.length).to.equal(30);
+    });
+  });
+
+  describe("event handling", () => {
+    it("attaches an event to data", () => {
+      const clickHandler = sinon.spy();
+      const wrapper = mount(
+        <VictoryScatter events={{data: {onClick: clickHandler}}}/>
+      );
+      wrapper.find(Point).forEach((node) => {
+        node.simulate("click");
+        expect(clickHandler.called).to.equal(true);
```

Comment:
In our discussion we decided that the event handler should be called with the point identifier (the `x` value or any other `key` / `id` used in the `data[]` object. Is that still the direction? Let's assert on that, too.